vue-app/src/views/Profile.vue

```diff
@@ -289,6 +343,29 @@ p.no-margin {
     .balances-card {
       padding: 0rem;
     }
+
+    .project-item {
+      display: flex;
+      padding: 1rem 0;
+      border-bottom: 1px solid rgba($highlight-color, 0.5);
+      .project-thumbnail {
+        width: 3rem;
+        aspect-ratio: 1 / 1;
+        object-fit: cover;
+        border-radius: 0.5rem;
+      }
+      .project-details {
+        flex: 1;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
```

Comment:
This is great. The only thing I was curious about some opinions on is whether we should `justify-conten`t as `space-between` or as `center`. I like how the title of the project is centered this way, but was wondering what some other peoples opinions on this might be. 
<img width="401" alt="Screen Shot 2021-08-29 at 8 07 34 PM" src="https://user-images.githubusercontent.com/15589226/131275649-d24ea6c7-6943-49e7-9b6f-2a79365db68d.png">
vs
<img width="400" alt="Screen Shot 2021-08-29 at 8 07 42 PM" src="https://user-images.githubusercontent.com/15589226/131275663-e3877cbf-b0ee-4eed-bb7f-25e982208bc5.png">

and also 

<img width="400" alt="Screen Shot 2021-08-29 at 8 07 07 PM" src="https://user-images.githubusercontent.com/15589226/131275678-90feb173-12e9-497c-b365-969d7cb85319.png">
vs
<img width="406" alt="Screen Shot 2021-08-29 at 8 06 58 PM" src="https://user-images.githubusercontent.com/15589226/131275691-5735152e-0b45-439b-a55d-87b15bb4e1db.png">

Other than that one question, this looks awesome!

Reply:
Not a strong stance stance here, though I suppose I prefer the `center` alignment.

Reply:
Yeah good question, wasn't sure myself. Went with space-between so if given a list of projects with some under review and some not, that would leave all the titles aligned to the top, rather than switching back and forth.

Reply:
Tested this out... doesn't look that bad at all actually, will switch to `center`